src/sample/Controller.java

```diff
@@ -1,4 +1,62 @@
 package sample;
 
-public class Controller {
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.fxml.FXML;
+import javafx.fxml.Initializable;
+import javafx.scene.control.ListView;
+import javafx.scene.control.TextField;
+import javafx.scene.input.KeyEvent;
+
+import java.net.URL;
+import java.util.ResourceBundle;
+
+public class Controller implements Initializable {
+    ObservableList<Contact> items = FXCollections.observableArrayList();
+
+
+    @FXML
+    ListView list;
+
+    @FXML
+    TextField name;
+
+    @FXML
+    TextField phone;
+
+    @FXML
+    TextField email;
+
+
+    public void addItems() {
+        if (items.isEmpty()){
+            return;
```

Comment:
Line 32 is incorrect.  It is saying that if the items array is empty return void.  The array will always be empty to start.  So the lines 35 down will NEVER RUN!